data/mods/Aftershock/lore/factions.md

```diff
@@ -27,7 +27,7 @@ Valid locations that could be added:
 - Small military outposts out in the frozen wastes.
 - Manned infrastructure buildings. (Power plants, radar stations, planetary defenses, smaller spaceports)
 
-Additionaly theres a single UICA space cruiser stationed on polar orbit.
+Additionally there's a single UICA space cruiser stationed on polar orbit.
```

Comment:
```suggestion
Additionally there's a single UICA space cruiser stationed in polar orbit.
```
'On Orbit' typically is used to refer to where an action, experiment, or operation is taking place. In other words, the space shuttle is described as being 'in orbit,' while the experiments conducted on the shuttle are referred to as being conducted 'on orbit.